functions/src/common/auth.ts

```diff
@@ -107,3 +107,10 @@ export function canImport(
   const role = getRole(user, survey);
   return !!role && [Pb.Role.SURVEY_ORGANIZER].includes(role);
 }
+
+export function canManage(
```

Comment:
It seems like this just calls `canImport`, maybe it makes sense to just call it directly instead?